### ISSUE TITLE
Generated Kotlin User.kt has compilation error on @OneToMany(cascade = CascadeType.ALL)

### ISSUE THREAD

**Issue by leozin** [jhipster/jhipster-kotlin#240]
## Overview of the issue
Generated Kotlin User.kt has a compilation error
In Kotlin, annotated-parameters which are Array need to be declared as an Array and not single element.
Eg.:

```
@JsonIgnore
@OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, mappedBy = "user")
var persistentTokens: MutableSet<PersistentToken> = mutableSetOf(),
```

The code above will fail to compile. The fix is simple, just add brackets to the cascade type:
`@OneToMany(cascade = [ CascadeType.ALL ], orphanRemoval = true, mappedBy = "user")`

Then it fixes the problem

## Motivation for or Use Case
No motivation. This is the code generated initially.

## Reproduce the error
Create a project using Khipster

## Link to PR (if any)
I will create a PR now linked to this issue with the solution. I couldn't find a page stating how we should name the PR, what's the process etc. Or maybe I'm just being lazy.

The solution seems to be simple, by just changing the User.kt.ejs template, here: https://github.com/jhipster/jhipster-kotlin/blob/7c94fd1d9f3566b35f40d73f66ecee94f6567cd8/generators/server/templates/src/main/kotlin/package/domain/User.kt.ejs#L363

And enclose the `CascadeType.ALL` with brackets.

## KHipster Version(s)
1.8.1

-   [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by sendilkumarn** [comment]
@leozin Please feel free to create a PR. There are no naming constraints, please feel free to use some meaningful commit message. Thanks :)

**Comment by leozin** [comment]
@sendilkumarn cool I will do that in the evening :) thanks for the quick reply

**Comment by sendilkumarn** [comment]
Closed by #242
